src/main/resources/hudson/plugins/promoted_builds/PromotionProcess/process-config.jelly

```diff
@@ -44,7 +44,7 @@
       <f:optionalBlock name="hasAssignedLabel" title="${%Restrict where this promotion process can be run}"
                        checked="${instance.assignedLabelString!=null}" inline="true">
         <f:entry title="${%Label Expression}"
-                 description="If not set, the label of the promoted build will be used.">
+                 description="If not set, the label of the promoted build will be used." field="assignedLabelString">
```

Comment:
Why was this needed?

Reply:
Yes, it should not ne needed

Reply:
without the `field` the help message is not displayed at all (no ? button on the right)

Reply:
I wonder if this is a bug with an optional block or something - should not be needed as this should be on the textbox field below.

Reply:
IMO this is correct (the help link is determined from `f:entry`, however the textbox can be simplified now to `<f:textbox autoCompleteDelimChar=" "/>`

Reply:
ok let me change it then.